test/src/userobjects/RestartableTypes.C

```diff
@@ -17,7 +17,7 @@
 template<>
 InputParameters validParams<RestartableTypes>()
 {
-  InputParameters params = validParams<UserObject>();
+  InputParameters params = validParams<GeneralUserObject>();
```

Comment:
They are related. By adding parameters to the interface, which is on GeneralUserObject, these objects miss getting the new parameter.

Even if we don't add the parameter to the interface in the end, these should be fixed.